README.md

```diff
@@ -21,6 +21,10 @@ If you'd like to help out, please see [CONTRIBUTING.md](CONTRIBUTING.md).
 
 * Next release:
   - Planned: removal of unused signature verification schemes at link-time.
+  - Planned: expose AlertDescription, ContentType, and HandshakeType,
```

Comment:
I think `Planned` is used for features that are not yet implemented, so it doesn't apply here.